---
site: freiburg
title: 'Galaxy Australia tool updates 2020-05-26'
tags: [tools]
supporters:
    - galaxyaustralia
    - melbinfo
    - qcif
---


<style>
  table {
    width: 100%;
    margin: 10px 20px;
  }
  table th {
    display: none;
  }
  td {
    padding: 3px 5px;
  }
  tr td:nth-child(1) {
    vertical-align: top;
    width: 25%;
  }
</style>

### Tools installed

| Section | Tool |
|---------|-----|
| **Annotation** | sortmerna [eb35257d2e29](https://toolshed.g2.bx.psu.edu/view/rnateam/sortmerna/eb35257d2e29)<br/>hmmer3 [54b6b4d0613a](https://toolshed.g2.bx.psu.edu/view/iuc/hmmer3/54b6b4d0613a) |
| **FASTA/FASTQ** | ucsc_fasplit [71f649c23021](https://toolshed.g2.bx.psu.edu/view/iuc/ucsc_fasplit/71f649c23021) |
| **Fetch Sequences/Alignments** | fraggenescan [1f0a3e4497d4](https://toolshed.g2.bx.psu.edu/view/iuc/fraggenescan/1f0a3e4497d4) |
| **Metagenomic Analysis** | maxbin2 [cfd50144a871](https://toolshed.g2.bx.psu.edu/view/mbernt/maxbin2/cfd50144a871)<br/>cat_bins [0094893f5001](https://toolshed.g2.bx.psu.edu/view/iuc/cat_bins/0094893f5001)<br/>interproscan5 [e32f2ea6a139](https://toolshed.g2.bx.psu.edu/view/bgruening/interproscan5/e32f2ea6a139) |
| **Proteomics** | maxquant_mqpar [3fc2116ac6d9](https://toolshed.g2.bx.psu.edu/view/galaxyp/maxquant_mqpar/3fc2116ac6d9) |
| **RNA-seq** | blockclust [aab6cf87b40a](https://toolshed.g2.bx.psu.edu/view/rnateam/blockclust/aab6cf87b40a) |
| **Variant Calling** | lofreq_alnqual [435b157bed6d](https://toolshed.g2.bx.psu.edu/view/iuc/lofreq_alnqual/435b157bed6d)<br/>lofreq_filter [9cb929f21cfa](https://toolshed.g2.bx.psu.edu/view/iuc/lofreq_filter/9cb929f21cfa)<br/>lofreq_indelqual [f4473557ffc7](https://toolshed.g2.bx.psu.edu/view/iuc/lofreq_indelqual/f4473557ffc7) |

### Tools updated

| Section | Tool |
|---------|-----|
| **BED** | bedtools [e19bebe96cd2](https://toolshed.g2.bx.psu.edu/view/iuc/bedtools/e19bebe96cd2) |
| **ChemicalToolBox** | rdock_rbcavity [744a777e9f90](https://toolshed.g2.bx.psu.edu/view/bgruening/rdock_rbcavity/744a777e9f90)<br/>rdock_rbdock [a22969b08177](https://toolshed.g2.bx.psu.edu/view/bgruening/rdock_rbdock/a22969b08177) |
| **FASTQ Quality Control** | fastqc [f2e8552cf1d0](https://toolshed.g2.bx.psu.edu/view/devteam/fastqc/f2e8552cf1d0) |
| **Graph/Display Data** | plotly_ml_performance_plots [85da91bbdbfb](https://toolshed.g2.bx.psu.edu/view/bgruening/plotly_ml_performance_plots/85da91bbdbfb)<br/>plotly_parallel_coordinates_plot [7b21a9b5922f](https://toolshed.g2.bx.psu.edu/view/bgruening/plotly_parallel_coordinates_plot/7b21a9b5922f)<br/>circos [4b519282a05b](https://toolshed.g2.bx.psu.edu/view/iuc/circos/4b519282a05b) |
| **Machine Learning** | sklearn_ensemble [1a53edc4b438](https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_ensemble/1a53edc4b438)<br/>sklearn_searchcv [25ab6b4e376e](https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_searchcv/25ab6b4e376e)<br/>sklearn_svm_classifier [f9639b488779](https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_svm_classifier/f9639b488779)<br/>sklearn_generalized_linear [91c4e3331208](https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_generalized_linear/91c4e3331208) |
| **Mapping** | bwa [3fe632431b68](https://toolshed.g2.bx.psu.edu/view/devteam/bwa/3fe632431b68) |
| **RNA-seq** | trinity_define_clusters_by_cutting_tree [55292a66e7c1](https://toolshed.g2.bx.psu.edu/view/iuc/trinity_define_clusters_by_cutting_tree/55292a66e7c1) |
| **SAM/BAM** | qualimap_bamqc [19ece8afbaab](https://toolshed.g2.bx.psu.edu/view/iuc/qualimap_bamqc/19ece8afbaab) |
| **Single-cell** | scanpy_filter [3c86f71498bc](https://toolshed.g2.bx.psu.edu/view/iuc/scanpy_filter/3c86f71498bc)<br/>scanpy_normalize [a407e7f8bdc1](https://toolshed.g2.bx.psu.edu/view/iuc/scanpy_normalize/a407e7f8bdc1)<br/>scanpy_remove_confounders [215733a383f2](https://toolshed.g2.bx.psu.edu/view/iuc/scanpy_remove_confounders/215733a383f2) |
| **Statistics** | sklearn_build_pipeline [840f29aad145](https://toolshed.g2.bx.psu.edu/view/bgruening/sklearn_build_pipeline/840f29aad145) |
